Task3/EEL_task_3/src/main/java/com/epam/esm/giftcertificate/service/GiftCertificateService.java

```diff
@@ -49,10 +50,10 @@ public GiftCertificate createCertificate(GiftCertificate giftCertificate) {
 
     }
 
-    public boolean deleteCertificate(long id) {
+    public HttpStatus deleteCertificate(long id) {
```

Comment:
Service shouldn't know about HttpStatus, please return model or void and throw exception in error cases

Reply:
Ready